cmd/fuzzbinder/main.go

```diff
@@ -42,6 +42,8 @@ func main() {
 		"Required - Fuzzing date. The expected date format is YYYYMMDD.")
 	fuzzClaimPath := flag.String("fuzzclaim_path", "fuzzclaim.json",
 		"Optional - Output file name for storing the generated fuzzing claim.")
+	validityDays := flag.Int("validity", 7,
+		"Optional - Validity period of the fuzzing claim that will be generated in days.")
```

Comment:
Maybe add in the comment that the default is 7 days?

Reply:
In this case I need to add the defaults to all the other comments for consistency. Do I do it in this PR?

Reply:
No need to do this in this PR. Good idea, actually, to add comments for all in the same PR!

I do think that the command line providing the default values is nice to have. What do you think, @rbehjati ?

Reply:
I agree that it is nice to have the defaults, but I'd check if the help already prints that. @MarahGamdou can you check and add the defaults only if the help does not already print them?

But I am not sure if 7 days is a reasonable default. As I described above, I think it is better to have flags for `notBefore` and `notAfter` and only make `notBefore` optional.

Reply:
Yes, the help already adds the default. I checked it.

Reply:
Thanks for checking :)